frappe/utils/jinja.py

```diff
@@ -57,16 +57,39 @@ def render_template(template, context, is_path=None):
 	:param context: dict of properties to pass to the template
 	:param is_path: (optional) assert that the `template` parameter is a path'''
 
+	from frappe import _, get_doc
+	from frappe.model.document import Document
+
 	if not template:
 		return ""
 
 	# if it ends with .html then its a freaking path, not html
-	if (is_path
+	_doc = None
+
+	if isinstance(context, Document):
+		_doc = context
+	elif isinstance(context, dict):
+		if "doc" in context:
+			_doc = context["doc"]
+		elif "doctype" in context and "docname" in context:
+			_doc = get_doc(context["doctype"], context["docname"])
+	elif hasattr(context, "doc"):
+		_doc = context.doc
+	elif hasattr(context, "doctype") and hasattr(context, "docname"):
+		_doc = get_doc(context.doctype, context.docname)
+
+	if _doc:
+		_doc = get_doc(_doc.as_dict(translated=True))
+		_doc.flags.in_print = True
```

Comment:
Why this? I commented this out and didn't make a difference.

Reply:
Also _doc is not used anywhere in this method. Why is it required?

Reply:
Also, `_doc` is not used anywhere in this method.

Reply:
@netchampfaris It's make difference in terms of enable the translation of print formats

This `_doc` is the translated version of the original doc, by translated I mean, where the content fields, that the user enabled the translation, have been translated